Packs/RubrikPolaris/IncidentFields/incidentfield-Rubrik_Polaris_Message.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Rubrik Radar"
+    ],
+    "caseInsensitive": true,
+    "cliName": "rubrikpolarismessage",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": true,
+    "id": "incident_rubrikpolarismessage",
```

Comment:
this might be a problem as it has the same id as in `Packs/RubrikPolaris/IncidentFields/incidentfield-RubrikPolaris_Message.json`

Reply:
Thank you @roysagi!  I will try to resolve the id conflict for this particular field.  I know we ran into some validation problems earlier, but will try to get through this one.  I'll post any issues if they come up.

Reply:
@roysagi So we did find an accidental dependency from our Radar layout that incorrectly pointed to one of the "rubrikpolarismessage" incident fields.  We have replaced that dependency and have redirected it to the proper field which is named something entirely different with a totally different id (rubrikpolarisanalysispipeline).  So we are not using the following fields anymore which are the ones with the conflicting id's:

*  incidentfield-Rubrik_Polaris_Message
*  incidentfield-RubrikPolaris_Message

We would LOVE to delete these and never use them again.  My question is how can we do this?  We have validation challenges if we:

A.  Try to delete these field files and remove their entries from the mapping file.
B.  Try to change the id of one of these fields.

Is there a special way that we can force the validator to turn a blind eye and have these fields removed as if they never existed?  Or can we possibly change the id of one of them somehow?  These should not be used by the integration at all however and we would love to just have them deleted if possible.

I will also echo this in our Slack channel if that makes the conversation a little easier as well.

Reply:
The newer file have been deleted.